faqs/galaxy/datasets_troubleshooting_custom_genome.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Troubleshooting Custom Genome
+area: datasets    
+box_type: tip        
+layout: faq        
+contributors: [jennaj, nurzhamalyrys] 
+---
+
+
+If a custom genome dataset is producing errors, double check the format and that the chromosome identifiers between **ALL** inputs. Clicking on the bug icon {% icon galaxy-bug %} will often provide a description of the problem. This does not automatically submit a bug report, and it is not always necessary to do so, but it is a good way to get more information about why a job is failing.
+
+1. Custom genome not assigned as FASTA format
```

Comment:
looks like these steps are possible problems and not actions to be taken?

Reply:
> looks like these steps are possible problems and not actions to be taken?

Yes, here are possible problems that may happen described. Do you suggest me to change title from "troubleshooting" to "possible issues"?

Reply:
We can leave the title, but maybe have bullet points instead of numbers that suggest sequential steps?

Reply:
> We can leave the title, but maybe have bullet points instead of numbers that suggest sequential steps?

@beatrizserrano yes, then I will change to bullet points.